thirdparty/onnx/CMakeLists.txt

```diff
@@ -59,6 +59,12 @@ if(CMAKE_COMPILER_IS_GNUCXX AND LINUX AND CMAKE_CXX_COMPILER_VERSION VERSION_LES
     string(REPLACE "std::filesystem::path proto_u8_path = std::filesystem::u8path(proto_path);" "" CONTENT "${CONTENT}")
     string(REPLACE "proto_u8_path" "proto_path" CONTENT "${CONTENT}")
     file(WRITE onnx/onnx/common/file_utils.h "${CONTENT}")
+
+    file(READ onnx/onnx/defs/traditionalml/utils.h CONTENT)
```

Comment:
maybe we can disable ONNX FE for compilers which  don't support onnx requirements?

Reply:
I think we will be obligated do it later. Right now, as you can see, they only started moving to a modern syntax. I strongly suggest to have a wa for this version (to satisfy more customers), but when they will introduce more misalignment - I think it will be only one option for us, or move to a branch.